src/main/java/assets/css/AcesUp.css

```diff
@@ -1,65 +1,113 @@
+body{
+    background-image: url("../images/background.jpg");
+    background-size: 100%;
```

Comment:
This background code seems a little arbitrary with both background size and height, may want to check into this in the future